images/build/Dockerfiles/ltsVersions.Dockerfile

```diff
@@ -208,7 +202,8 @@ ARG SDK_STORAGE_ENV_NAME
 ARG SDK_STORAGE_BASE_URL_VALUE
 WORKDIR /
 
-ENV PATH="$PATH:/opt/oryx:/opt/nodejs/lts/bin:/opt/dotnet/sdks/lts:/opt/python/latest/bin:/opt/yarn/stable/bin:/opt/hugo"
+ENV ORIGINAL_PATH="$PATH"
+ENV PATH="/opt/oryx:/opt/nodejs/lts/bin:/opt/dotnet/sdks/lts:/opt/python/latest/bin:/opt/yarn/stable/bin:/opt/hugo:$PATH"
```

Comment:
does it matter if you have $PATH before or after?

Reply:
Yes it does. Since here we want Oryx installed tools to take preference by default, where as in case of VSO image we would have Oryx installed paths later to enable users to pick their own installed versions